### ISSUE TITLE
Deny Access to All Except Logged In Users

### ISSUE THREAD

**Issue by CyrusVatandoost** [CyrusVatandoost/warehouse#49]
If the user isn't logged in, redirect the user to go to the login page.

**Comment by christalon** [comment]
what pages exactly?

**Comment by CyrusVatandoost** [comment]
Everything except login/register/welcome im guessing

**Comment by christalon** [comment]
i just realized, i think this should be pulled later since the others would have a hard time editing the ui and such

**Comment by CyrusVatandoost** [comment]
Wait. I made a mistake sorry.

As a guest you are not allowed to see the register page because the site would be exclusive to to T3D diba? So only the admin can create accounts and it's up to those registered accounts to edit their own preferences.

So as a guest (not logged in), the site will not show the Register page for now. I'll update what else to change later.

**Comment by christalon** [comment]
ok

**Comment by CyrusVatandoost** [comment]
I made a mistake. We made the decision to allow anyone to register however they cannot access the site until the admin will approve their application. So as a guest, you can see the register page. Basically everything na as of the moment really sorry :( 

I'll make a new issue about approving applications

**Comment by christalon** [comment]
I thought of something, an admin can send a user a signup link once they were approved, just a suggestion :)

**Comment by christalon** [comment]
although that link should be unique and could be used only once

**Comment by CyrusVatandoost** [comment]
Right we can also add that. Like a generated register link? or generated complete registration link?

**Comment by christalon** [comment]
we could also do this http://bensmith.io/email-verification-with-laravel

**Comment by CyrusVatandoost** [comment]
Make a new issue for that nalang

**Comment by christalon** [comment]
ok sige :)
